apps/api-extractor/src/generators/DtsRollupGenerator.ts

```diff
@@ -110,15 +110,25 @@ export class DtsRollupGenerator {
       }
```

Comment:
Is this check still needed?  It seems redundant.

Reply:
I thought about this when adding... I figured keeping the old behaviour was alright (rolls up the comments to one symbol when they're all excluded) however, it makes more sense to have it consistent. I'll make the change